docs/book/user-guide/advanced-guide/data-management/model-management.md

```diff
@@ -204,6 +209,7 @@ particular model version. Possible options for stages are:
 - `latest`: The latest version of the model. This is a virtual stage to retrieve the latest version only - versions cannot be promoted to `latest`.
 - `archived`: This is archived and no longer relevant. This stage occurs when a
   model moves out of any other stage.
+  > is this archived or "No stage"?
```

Comment:
It is Archived, we never fall back to no stage.

Reply:
okay! i just noticed "no stage" on the dashboard for older versions, so I asked.